.github/ISSUE_TEMPLATE/02_questions.md

```diff
@@ -1,17 +1,10 @@
 ---
 name: "\U00002753 Questions?"
-about: If you have *specific* questions about the website, please post them here.
+about: This issue template should not be used anymore!
```

Comment:
I'd suggest to put more info on where questions were moved to into the "about" text. In its current state, people would only find out if they proceed with this template. Unfortunately, it seems like you can't put links in there

```suggestion
about: This issue template should not be used anymore! Please go to Discussions and create a new discussion choosing the Category "Q&A".

```